tunnel-server/src/auth.ts

```diff
@@ -101,91 +101,72 @@ export const jwtAuthenticator = (
   const parsedJwt = decodeJwt(jwt)
   if (parsedJwt.iss === undefined) throw new AuthError('Could not find issuer in JWT')
 
-  let verificationData: VerificationData
-  try {
-    verificationData = getVerificationData(parsedJwt.iss, publicKeyThumbprint)
-  } catch (e) {
-    if (e instanceof AuthError) return { isAuthenticated: false }
-
-    throw e
+  const jwtIssuer = issuers.find(x => x.issuer === parsedJwt.iss)
```

Comment:
probably should add a log here

Reply:
Currently it's the same behavior is before. (previous, it throw AuthError which is specially ignore)
Do we have a static logger here? If not, I'm not sure it's worth adding at this level. 
Since this method already throwing and having return value, it can delegate it externally without adding additional dependecy. (which can also be useful information for the app to decide whether to use 400/401)